src/styles/atoms/_atoms.button.scss

```diff
@@ -643,6 +643,7 @@
 .a-button {
   &.has-avatar,
   &.has-avatar-with-image {
+    overflow: hidden;
```

Comment:
Waar dient dit voor?

Reply:
Als de border-radius van de button op, bijvoorbeeld, 4px gezet wordt, dan gaan we nog steeds de rechte hoeken van de avatar te zijn krijgen. Met die `overflow: hidden;` is dit opgelost.

<img width="407" alt="Screenshot 2021-10-15 at 08 08 36" src="https://user-images.githubusercontent.com/4232875/137440393-d1ac3dde-34c6-40b9-a56d-e93db723775b.png">.

Zonder die `overflow: hidden;` zou je linksbovenaan en linksonderaan rechte hoeken krijgen. 😄